README.md

```diff
@@ -2,16 +2,39 @@
 
 [![CircleCI](https://circleci.com/gh/invainn/ARIA/tree/master.svg?style=svg)](https://circleci.com/gh/invainn/ARIA/tree/master)
 
-## Description
+## Project Description
 ---
-TODO
+ARIA (Administration, Registration, and Information Assistant) is a web application solution for the Northern Nevada Music Teachers Association (NNMTA). Previous versions of ARIA were hosted on WordPress and contained performance issues as well as functional problems with certain features for both customers and adminstrators.
+
+This version of ARIA (version 3.0) attempts to solve problems with previous iterations of ARIA by creating a full-stack web application built in React.js, Node.js, and PostgreSQL, along with many other tools. Building the web application from the ground up allows our team to have more control over development and addressing the needs of the customer.
+
+
+This project is being developed to satisfy the CS 425 - Software Engineering course requirements at the University of Nevada Reno. Our sponsoring faculty member from the Computer Science and Engineering Department is Dr. Frederick Harris, and our Ph.D. advisor/mentor is Jalal Kiswani.
 
 ## Team Members
 ---
 * Anthony Bennett
 * Nikkolas Irwin
 * Kevin Carlos
 
+## Running the application
```

Comment:
Need to add more detailed instructions as building the image, what to do when `package.json` is updated, and how development works by using container volumes